spyder/plugins/explorer/widgets/main_widget.py

```diff
@@ -341,13 +344,16 @@ def setup(self):
 
         for action in [
             self.filter_button,
+            self.remote_treewidget.upload_file_action,
             self.refresh_action,
         ]:
             self.add_corner_widget(action, before=self._options_button)
 
     def update_actions(self):
         """Handle the update of actions of the plugin."""
-        pass
+        enable_remote_actions = self.get_current_widget() == self.remote_treewidget
```

Comment:
```suggestion
        enable_remote_actions = (
            self.get_current_widget() == self.remote_treewidget
        )
```

Make text fit in 79 columns.